<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VLS - SceneEditor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/img/vls.png</normaloff>:/res/img/vls.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="RenderWidget" name="widget" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>360</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>Effect</string>
    </property>
    <addaction name="actionAdd_shader"/>
    <addaction name="actionReload"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuRender">
    <property name="title">
     <string>Render</string>
    </property>
    <addaction name="actionReset_camera"/>
    <addaction name="actionTake_screenshot"/>
    <addaction name="actionProjector_output"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScene"/>
   <addaction name="menuRender"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/new.png</normaloff>:/res/img/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/open.png</normaloff>:/res/img/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/save.png</normaloff>:/res/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAdd_shader">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/new shader.png</normaloff>:/res/img/new shader.png</iconset>
   </property>
   <property name="text">
    <string>Add layer</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionReset_camera">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/camera.png</normaloff>:/res/img/camera.png</iconset>
   </property>
   <property name="text">
    <string>Reset camera</string>
   </property>
  </action>
  <action name="actionTake_screenshot">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/screenshot.png</normaloff>:/res/img/screenshot.png</iconset>
   </property>
   <property name="text">
    <string>Take screenshot</string>
   </property>
  </action>
  <action name="actionProjector_output">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/projector.png</normaloff>:/res/img/projector.png</iconset>
   </property>
   <property name="text">
    <string>Projector output</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/setting2.png</normaloff>:/res/img/setting2.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderWidget</class>
   <extends>QWidget</extends>
   <header>renderwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
